reproducibility_project/lrc_shift_subproject/src/engines/cassandra/project.py

```diff
@@ -0,0 +1,535 @@
+"""Setup for signac, signac-flow, signac-dashboard for this study."""
+# import foyer
+import os
+from pathlib import Path
+
+import flow
+import pandas as pd
+from flow import directives
+
+import reproducibility_project.templates.ndcrc
+
+
+class Project(flow.FlowProject):
+    """Subclass of FlowProject to provide custom methods and attributes."""
+
+    def __init__(self):
+        super().__init__()
+        current_path = Path(os.getcwd()).absolute()
+        self.data_dir = current_path.parents[1] / "data"
+        self.ff_fn = self.data_dir / "forcefield.xml"
+
+
+@Project.label
+def is_cassandra(job):
+    """Verify that the job is for Cassandra."""
+    return job.sp.engine == "cassandra"
+
+
+@Project.label
+def cassandra_complete(job):
+    """Check whether job is complete."""
+    return check_complete(job.fn("prod"))
+
+
+def check_complete(run_name):
+    """Check whether MoSDeF Cassandra simulation with run_name has completed."""
+    complete = False
+    fname = run_name + ".out.log"
+    if not os.path.exists(fname):
+        return complete
+    with open(fname) as f:
+        for line in f:
+            if "Cassandra simulation complete" in line:
+                complete = True
+                break
+    return complete
+
+
+@Project.operation
+@Project.pre(is_cassandra)
+@Project.post(cassandra_complete)
+@directives(omp_num_threads=8)
+def run_cassandra(job):
+    """Run all simulation stages for given statepoint."""
+    import foyer
+    import mbuild as mb
+    import mosdef_cassandra as mc
+    import numpy as np
+    import unyt as u
+    from mbuild.formats.xyz import read_xyz
+    from pymbar.timeseries import detectEquilibration
+
+    from reproducibility_project.src.molecules.system_builder import (
+        construct_system,
+        get_molecule,
+    )
+    from reproducibility_project.src.utils.forcefields import load_ff
+
+    # MoSDeF project - Cassandra keyword mappings
+
+    cass_ensembles = {
+        "NPT": "npt",
+        "GEMC-NVT": "gemc",
+    }
+
+    cass_cutoffs = {
+        ("hard", None): "cut",
+        ("hard", "energy_pressure"): "cut_tail",
+        ("shift", None): "cut_shift",
+        ("shift", "energy_pressure"): None,
```

Comment:
This isn't necessary since this will not be a statepoint, and if it were a statepoint, it would cause an error in Cassandra because `None` is not a valid cutoff style.

```suggestion
```

Reply:
@emarinri and @rwsmith7531 , once this suggestion is committed, this should be ready to merge, right?